doc/posts/zitification/ssh/zitifying-ssh.md

```diff
@@ -0,0 +1,133 @@
+# Zitifying SSH
+
+As we learned in the [opening post](../README.md) - "zitifying" an application means to embed a Ziti SDK into an
+application and leverage the power of
+the [Ziti Network](https://openziti.github.io/ziti/overview.html#overview-of-a-ziti-network) to provide secure, truly
```

Comment:
I read and reread the sentence because it still feels like it can go both ways for me. Any given identity is only able to access a single network which is how I arrived at using ' the network'. I do concede that given how it reads, "a" certainly seems to be able to fit. since i'm on the fence i'm gonna leave 'the' unless you have a good counterpoint? I'm happy to change it, I'm not beholden to 'the' here...

Reply:
i rearranged it a bit more - it'll be:

> When your application uses
> a [Ziti Network](https://openziti.github.io/ziti/overview.html#overview-of-a-ziti-network) configured
> with a truly zero-trust mindset, it will be impervious to the "expand/multiply"